healthtools/config.py

```diff
@@ -22,16 +22,16 @@
 SMALL_BATCH_HF = 100  # No of records scraped from health-facilities sites in development mode
 SMALL_BATCH_NHIF = 1  # No of nhif accredited facilities scraped in development mode
 
-# Where we archive the data
-DATA_DIR = os.getcwd() + "/data/"
-if not os.path.exists(DATA_DIR):
-    os.mkdir(DATA_DIR)
-    os.mkdir(DATA_DIR + "archive")
-    os.mkdir(DATA_DIR + "test")
-    os.mkdir(DATA_DIR + "test/archive")
-
 if AWS["s3_bucket"]:
     DATA_DIR = "data/"
+else:
+    # Where we archive the data in case of no s3 Bucket
+    DATA_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__))) + "/data/"
+    if not os.path.exists(DATA_DIR):
+        os.mkdir(DATA_DIR)
+        os.mkdir(DATA_DIR + "archive")
+        os.mkdir(DATA_DIR + "test")
+        os.mkdir(DATA_DIR + "test/archive")
```

Comment:
We have been running on the assuming that everyone knows the structure of the folders and will have set it up before running. We should definitely do the same for aws folders